applications/radisys/readme.md

```diff
@@ -0,0 +1,80 @@
+**Radisys Engage Media Server**
+
+Radisys Corporation ©
```

Comment:
Correct syntax for copyright is explained in [this doc](https://github.com/open-ness/edgeapps/blob/master/DEVELOPING.md#license-headers).